### ISSUE TITLE
Failed to upgrade caused by `gnome.vte`

### ISSUE THREAD

**Issue by astrojhgu** [NixOS/nixpkgs#155549]
### Describe the bug
Failed to upgrade with `nixos-rebuild boot --upgrade` in unstable channel.

### Steps To Reproduce
```
sudo nixos-rebuild boot --upgrade
```

### Expected behavior
The upgrading should be proceeded normally

### Screenshots
```
$> sudo nixos-rebuild boot --upgrade
unpacking channels...
building Nix...
building the system configuration...
trace: warning: findutils locate does not support pruning by directory component
error: The ‘gnome.vte’ alias was removed on 2022-01-13. Please use ‘pkgs.vte’ directly.
(use '--show-trace' to show detailed location information)
```

### Additional context
Even though I had already removed all `gnome`-related items in my `/etc/nixos/configuration.nix`

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ sudo nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.91, NixOS, 22.05 (Quokka)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.5.1`
 - channels(root): `"nixos-22.05pre346355.d5dae6569ea"`
 - channels(astrojhgu): `"android, nixos-21.11pre329341.9e70e9f732f"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by teto** [comment]
try with --show-trace to pinpoint where it's at.

**Comment by astrojhgu** [comment]
> try with --show-trace to pinpoint where it's at.

I past the show trace result in the attached file

[trace.md](https://github.com/NixOS/nixpkgs/files/7897341/trace.md)

**Comment by gador** [comment]
I'm having the same issue. The trace is almost identical. It seems xfce4-terminal might play a role on it. It does, however, use `vte` and not `gnome.vte`

**Comment by mbrgm** [comment]
Same issue here. e703bc0ade3b6a3669f77d0acede1c2047e51fc2 appears to be the first bad commit.

**Comment by bobby285271** [comment]
Should be fixed in #155741
